tendermint/src/types/proto/info.rs

```diff
@@ -82,12 +82,18 @@ pub struct Evidence {
     /// The height when the offense occurred
     pub height: i64,
     /// The corresponding time where the offense occurred
-    pub time: Option<Timestamp>,
+    pub time: Timestamp,
     /// Total voting power of the validator set in case the ABCI application does
     /// not store historical validators.
     pub total_voting_power: i64,
 }
 
+impl Evidence {
+    pub fn r#type(&self) -> EvidenceType {
```

Comment:
I've followed the name of field and code in sdk. I think that it is better to save the name here because it's ibc compatible struct. 
@joneskm  what do you think?

Reply:
I think it's common to rename type into kind in rust so you should be fine.

Reply:
I don't think renaming will cause any compatibility problems